pkgs/development/libraries/vte/default.nix

```diff
@@ -61,6 +61,7 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     fribidi
+    gobject-introspection
```

Comment:
Possibly because of https://github.com/NixOS/nixpkgs/pull/177538#discussion_r896125213